lib/Supervisor.js

```diff
@@ -6,7 +6,6 @@ const fs = require('fs');
 const net = require('net');
 const split = require('split');
 const XmlBuilder = require('xmlbuilder');
-// $FlowFixMe: Flow marks this line as an error (and only in this file!) but then it knows the types of `Report` anyway. This is the “error”: Cannot build a typed interface for this module. You should annotate the exports of this module with types. Missing type annotation at declaration of variable `Report`: [signature-verification-failure]
```

Comment:
This stopped being a problem when I improved .flowconfig. I have no idea why, but I’m not complaining.

Reply:
nice, I won't complain either :)